samcli/local/docker/lambda_build_container.py

```diff
@@ -298,4 +308,8 @@ def get_image_tag(architecture):
         str
             Image tag
         """
+        if architecture not in [X86_64, ARM64]:
```

Comment:
Can we re-use the `_validate_architecture` function above (just made public)?

Reply:
Oh, yes, good point!